zerver/openapi/zulip.yaml

```diff
@@ -4183,6 +4183,11 @@ paths:
                                         methods are enabled for the organization. For authentication UI itself,
                                         clients should use the pre-authentication metadata returned by
                                         [`GET /server_settings`](/api/get-server-settings).
+
+                                        **Changes**: In Zulip 9.0 (feature level 243), the
+                                        format was changed to include information, per authentication method,
+                                        about not only whether it is enabled, but also whether it can be used
+                                        by the organization on Zulip Cloud, based on the active plan.
```

Comment:
I rewrote this and the other OpenAPI descriptions in this commit, for a couple reasons.

One is that these should always explain what the **previous** behavior was, because that information is otherwise unavailable. And the second is that we don't really need to say what the new format's values are -- those are right there in the documentation.